.github/workflows/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM centos as centos-python
 RUN yum install -y make gcc openssl-devel libffi-devel sqlite-devel
 ARG PYTHON_VER
 ENV LC_CTYPE="en_US.UTF-8"
+ENV LANG C.UTF-8
```

Comment:
I'm curious about this one.  What does it do?  I mean, there already was a LC_CTYPE=en_US.UTF-8, so locale charset was UTF-8. Did you find some translated messages messing things up?

Reply:
this was just a hunch trying to fix the [last issue](https://travis-ci.org/github/buildout/buildout/jobs/728115069#L712) which looks to me like the terminal here isn't set to unicode. If you have a better idea I'll be happy to try it

Reply:
Actually, it was a good try!  The `centos` docker image doesn't support `en_US.UTF-8`, but `C.UTF-8` works.

The only problem is LC_CTYPE overrides LANG, so your workaround wasn't actually being used.  Try setting LC_ALL instead of LANG and LC_CTYPE.